Empire/Empire.cpp

```diff
@@ -1506,8 +1506,8 @@ std::shared_ptr<const UniverseObject> Empire::Source() const {
 
 std::string Empire::Dump() const {
     std::string retval = "Empire name: " + m_name +
-                         " ID: "+ boost::lexical_cast<std::string>(m_id) +
-                         " Capital ID: " + boost::lexical_cast<std::string>(m_capital_id);
+                         " ID: "+ std::to_string(m_id) +
```

Comment:
When those lines are touched anyway you can also add the missing space between the string literal and the `+`.